### PR TITLE
test, no command response

### PR DESCRIPTION
**Pull request by whoastonic**
expecting an error if the response type of the command is none...

### CODE REVIEW

**Comment by exoRift** [comment]
? Why should it throw if there's no response? There are plenty of bots that either shouldn't respond due to the design of the command or respond by reacting with a check on the command message

**Comment by whoastonic** [comment]
> ? Why should it throw if there's no response? There are plenty of bots that either shouldn't respond due to the design of the command or respond by reacting with a check on the command message

I dunno

**Comment by exoRift** [comment]
Just fix the typing

File: index.d.ts [hunk 1]
```diff
@@ -65,7 +65,7 @@ declare module 'eris-boiler' {
 
   type CommandResults = MessageData | Promise<MessageData>
 
-  type MessageData = string | {
+  type MessageData = void | string | {
```
Comment (alex-taxiera) [inline index.d.ts:68]: move this void to `CommandResults`

Comment (alex-taxiera) [reply index.d.ts:68]: also it should be undefined not void

File: lib/orator/index.test.js [hunk 2]
```diff
@@ -332,6 +332,28 @@ test('hasPermission/dm command in guild', async (t) => {
   t.false(ok)
 })
 
+test('_tryToExecute/no response', async (t) => {
```
Comment (alex-taxiera) [inline lib/orator/index.test.js:335]: try to execute doesn't care about the command response, lets run a test on process command response to make sure it doesn't do anything when the command response is undefined

File: index.d.ts [hunk 3]
```diff
@@ -63,7 +63,7 @@ declare module 'eris-boiler' {
     msg: Message<PrivateTextableChannel>
   }
 
-  type CommandResults = MessageData | Promise<MessageData>
+  type CommandResults = void | MessageData | Promise<MessageData>
```
Comment (alex-taxiera) [inline index.d.ts:66]: `void` should be `undefined`

**Pull request merged**